### PR TITLE
SJQ2T-106: BE - Endpoint for remove user from company

### PR DESCRIPTION
**Pull request by MetoMladenovski**
Implemented remove company for user.

### CODE REVIEW

**Review by ivanfhtec — COMMENTED** [review]
looks good

File: backend/src/main/java/com/htecgroup/skynest/service/impl/UserServiceImpl.java [hunk 1]
```diff
@@ -198,11 +200,20 @@ public void demoteUser(
...
   @Override
-  public void addCompanyForUser(
-      @Valid @UserNotInACompany UUID uuid, @Valid @CurrentUserIsInCompany UUID companyId) {
-    CompanyDto companyDto = companyService.findById(companyId);
+  public void addCompanyForUser(@Valid @UserNotInACompany @UserNotAdmin UUID uuid) {
+    CompanyEntity companyEntity =
+        currentUserService
+            .getCompanyEntityFromLoggedUser()
+            .orElseThrow(UserNotInAnyCompanyException::new);
     UserDto userDto = findUserById(uuid);
```
Comment (DraganMitrasinovicHtec) [inline backend/src/main/java/com/htecgroup/skynest/service/impl/UserServiceImpl.java:207]: Maybe use the regular find (repo method) here, so we don't have to map from Dto to UserEntity.

**Review by ivanfhtec — APPROVED** [review]
looks good

File: backend/src/main/java/com/htecgroup/skynest/service/impl/UserServiceImpl.java (hunk 1 above)
Comment (MetoMladenovski) [reply backend/src/main/java/com/htecgroup/skynest/service/impl/UserServiceImpl.java:207]: I think I understand you. Let me see
I am keeping the findUserById from our service, to be consistent. Just reordered the mapping and I think I did what u wanted.

Comment (DraganMitrasinovicHtec) [reply backend/src/main/java/com/htecgroup/skynest/service/impl/UserServiceImpl.java:207]: Yep, I approve

**Review by DraganMitrasinovicHtec — APPROVED** [review]
I approve!

**Pull request merged**